From bdfcddd4f135dc1d4fefb0373d9ebb93ef5110cb Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Mon, 7 Dec 2015 16:41:19 -0500
Subject: [PATCH] trace-cmd: Fix trace-record.c with non ptrace compile

Fix ptrace_wait() when NO_PTRACE=1 is defined.

Status: upstream

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
[Romain backport to 2.6]
Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 trace-record.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trace-record.c b/trace-record.c
index df82883..f83ec4b 100644
--- a/trace-record.c
+++ b/trace-record.c
@@ -1099,7 +1099,7 @@ static void ptrace_wait(enum trace_type type, int main_pid)
 		 (!WIFEXITED(status) || pid != main_pid));
 }
 #else
-static inline void ptrace_wait(int main_pid) { }
+static inline void ptrace_wait(enum trace_type type, int main_pid) { }
 static inline void enable_ptrace(void) { }
 static inline void ptrace_attach(int pid) { }
 
-- 
2.5.5

